modules/distr-tracing-tempo-config-default.adoc

```diff
@@ -0,0 +1,151 @@
+// Module included in the following assemblies:
+//
+// * distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_content-type: REFERENCE
+[id="distr-tracing-tempo-config-default_{context}"]
+= Distributed tracing default configuration options
+
+The Tempo custom resource (CR) defines the architecture and settings to be used when creating the {TempoShortName} resources. You can modify these parameters to customize your {TempoShortName} implementation to your business needs.
+
+.Generic Tempo YAML
```

Comment:
See [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=elements-files-directories) suggest following the file type with the word _file_
```suggestion
.Generic Tempo YAML file
```

Reply:
Thank you. Applying manually.